registry/storage/catalog.go

```diff
@@ -95,3 +95,23 @@ func (reg *registry) Enumerate(ctx context.Context, ingester func(string) error)
 	return nil
 
 }
+
+func pathGreaterThan(pathX, pathY string) (b bool) {
+	splitPathX := strings.SplitN(pathX, "/", 2)
+	splitPathY := strings.SplitN(pathY, "/", 2)
+
+	if splitPathX[0] == splitPathY[0] {
+		if len(splitPathX) == 1 && len(splitPathY) == 1 {
+			return false
+		} else if len(splitPathX) == 1 && len(splitPathY) != 1 {
+			return false
+		} else if len(splitPathX) != 1 && len(splitPathY) == 1 {
+			return true
+		}
+
+		return pathGreaterThan(splitPathX[1], splitPathY[1])
```

Comment:
It does. splitPathX[1] contains the second part as I split it only into 2 part

see: https://play.golang.org/p/zXf4_5_-vz